### PR TITLE
Use parity u256

### PR DESCRIPTION
**Pull request by kingoflolz**
Use parity u256 type for better interoperability
Note: The serialized form has changed from base 10 to hex prefixed with 0x

### CODE REVIEW

File: rita/src/rita_client/exit_manager/mod.rs [hunk 1]
```diff
@@ -159,22 +160,34 @@ impl Handler<Tick> for ExitManager {
...
-                ExitState::GotInfo | ExitState::Pending => match exit_setup_request(&k) {
-                    Ok(_) => {
-                        info!("exit request to {} was successful", k);
-                    }
-                    Err(e) => {
-                        info!("exit request to {} failed with {:?}", k, e);
-                    }
-                },
+                ExitState::New => {
+                    Arbiter::handle().spawn(exit_general_details_request(k.clone()).then(
```
Comment (jkilpatr) [inline rita/src/rita_client/exit_manager/mod.rs]: what does this arbiter do?

File: rita/src/rita_common/http_client/mod.rs [hunk 2]
```diff
@@ -67,66 +43,101 @@ impl Message for Hello {
 impl Handler<Hello> for HTTPClient {
     type Result = ResponseFuture<LocalIdentity, Error>;
     fn handle(&mut self, msg: Hello, _: &mut Self::Context) -> Self::Result {
-        Box::new(self.executors.send(msg).then(|r| r.unwrap()))
+        info!("sending {:?}", msg);
+
+        let stream = TokioTcpStream::connect2(&msg.to);
+
+        let endpoint = format!("http://[{}]:{}/hello", msg.to.ip(), msg.to.port());
```
Comment (jkilpatr) [inline rita/src/rita_common/http_client/mod.rs]: did this sneak in from the request pull?

File: num256/Cargo.toml [hunk 3]
```diff
@@ -9,4 +9,5 @@ num = "0.1.41"
 serde = "1.0.24"
 serde_derive = "1.0.24"
 serde_json = "1.0.8"
-lazy_static = "1.0.0"
\ No newline at end of file
+lazy_static = "1.0.0"
+ethereum-types = {git="https://github.com/paritytech/primitives.git"}
```
Comment (drozdziak1) [inline num256/Cargo.toml]: We should probably be using a release of this, they have a fairly fresh `v0.3.2` tag (3 weeks old). What do you think?

Comment (kingoflolz) [reply num256/Cargo.toml]: Our changes were merged in 7 days ago, so unfortunately not released yet

Comment (drozdziak1) [reply num256/Cargo.toml]: Fair enough, approving

**Pull request merged**